pkg/snmp/snmpparse/config_snmp.go

```diff
@@ -59,6 +61,38 @@ func ParseConfigSnmp(c integration.Config) []SNMPConfig {
 
 	return snmpconfigs
 }
+func ParseConfigSnmpMain() []SNMPConfig {
+	snmpconfigs := []SNMPConfig{}
+	configs := []snmplistener.Config{}
+	//the UnmarshalKey stores the result in mapstructures while the snmpconfig is in yaml
+	//so for each result of the Unmarshal key we storre the result in a tmp SNMPConfig{} object
+	err := config.Datadog.UnmarshalKey("snmp_listener.configs", &configs)
+	if err != nil {
+		fmt.Printf("unable to get snmp config from snmp_listener: %v", err)
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏Since there is an error, I think we can return `nil` directly here and avoid running the rest of the code.